apps/ui-playground/content/design/components/alert.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Alert
+airtableId: recnvwaXqOucqLo9Y
```

Comment:
do we still depend on airtable?

Reply:
No - keeping here incase we add back the integration in the future